controllers/dnspolicy_controller_multi_cluster_test.go

```diff
@@ -183,148 +183,6 @@ var _ = Describe("DNSPolicy Multi Cluster", func() {
 
 	Context("loadbalanced routing strategy", func() {
 
-		Context("weighted", func() {
```

Comment:
Irrelevant, you can't have weighted without geo.

Reply:
i'm removing that one in #549